addons/mass_mailing/tests/test_mailing_list.py

```diff
@@ -79,21 +79,6 @@ def test_mailing_contact_to_list(self):
         subaction = action["params"]["next"]
         self.assertEqual(subaction["type"], "ir.actions.act_window_close")
 
-        # set mailing list, add contacts and redirect to mailing view
-        mailing2 = self.env['mailing.list'].create({
-            'name': 'Contacts Sublimator',
-        })
-
-        wizard_form.mailing_list_id = mailing2
-        wizard = wizard_form.save()
-        action = wizard.action_add_contacts_and_send_mailing()
-        self.assertEqual(contacts.list_ids, mailing + mailing2)
-        self.assertEqual(action["type"], "ir.actions.client")
-        self.assertTrue(action.get("params", {}).get("next"), "Should return a notification with a next action")
-        subaction = action["params"]["next"]
-        self.assertEqual(subaction["type"], "ir.actions.act_window")
-        self.assertEqual(subaction["context"]["default_contact_list_ids"], [mailing2.id])
```

Comment:
Seems weird to remove this part of the test instead of updating it. Am I missing something?